VideoSampleCaptureRender/ViewController.swift

```diff
@@ -215,11 +223,15 @@ class ViewController: UIViewController, UITextFieldDelegate {
     func videoCaptureConstraints () -> TWCVideoConstraints {
         /* Video constraints provide a mechanism to capture a video track using a preferred frame size and/or frame rate.
         
-        Here, we set the captured frame size to 960x540. Check TWCCameraCapturer.h for other valid video constraints values.
-        
-        960x540 video will fill modern iPhone screens. However, older devices (< iPhone 5s, iPad Air 2) will have trouble rendering this quality. */
+         Here, we set the captured frame size to 960x540. Check TWCCameraCapturer.h for other valid video constraints values. 
+         
+         960x540 video will fill modern iPhone screens. However, older devices (< iPhone 5s, iPad Air 2) will have trouble rendering 960x540 quality. So, we set captured frame size to 480x360 and frame rate to 15 for older devices. */
```

Comment:
Suggestion:

         960x540 video will fill modern iPhone screens. However, older 32-bit devices (A5, A6 based) will have trouble capturing, and encoding video at HD quality. For these devices we constrain the capturer to produce 480x360 video at 15fps. */